### PR TITLE
Add column position for paimon

### PR DESCRIPTION
**Pull request by zhangjun0x01**

### CODE REVIEW

File: docs/content/how-to/altering-tables.md [hunk 1]
```diff
@@ -127,6 +127,24 @@ ALTER TABLE my_table ADD COLUMNS (
...
+## Adding Column Position
+
+To add a new column with specified position, use FIRST or AFTER col_name.
+
+{{< tabs "add-column-position" >}}
+
+{{< tab "Spark3" >}}
+
+```sql
+ALTER TABLE my_table ADD COLUMN c FIRST;
```
Comment (JingsongLi) [inline docs/content/how-to/altering-tables.md:139]: No type?

**Review by JingsongLi — COMMENTED** [review]
Thanks @zhangjun0x01 for the contribution, just left one comment to documentation.

File: docs/content/how-to/altering-tables.md (hunk 1 above)
Comment (zhangjun0x01) [reply docs/content/how-to/altering-tables.md]: my mistake ,I update it

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**